scripts/prepare_discard_plate.py

```diff
@@ -98,14 +107,16 @@ def _run(self):
                 if artifact in step_associated_artifacts or has_workflow_stage(artifact, sample_discard_wf_stage_name):
                     self.info(
                         'Container %s might route because artifact %s in step_associated_artifacts (%s) or has been discarded before (%s)',
-                        container.name, artifact.name, artifact in step_associated_artifacts, has_workflow_stage(artifact, sample_discard_wf_stage_name)
+                        container.name, artifact.name, artifact in step_associated_artifacts,
+                        has_workflow_stage(artifact, sample_discard_wf_stage_name)
                     )
                 else:
                     # This container will have to wait
                     route_allowed = False
                     self.info(
                         "Container: %s won't route because artifact %s in step_associated_artifacts (%s) or has been discarded before (%s)",
-                        container.name, artifact.name, artifact in step_associated_artifacts, has_workflow_stage(artifact, sample_discard_wf_stage_name)
+                        container.name, artifact.name, artifact in step_associated_artifacts,
+                        has_workflow_stage(artifact, sample_discard_wf_stage_name)
                     )
             if route_allowed:
                 artifacts_to_route.extend(list(container.placements.values()))
```

Comment:
Print statement below is redundant and can be removed

Reply:
removed